hack/verify-golangci-lint.sh

```diff
@@ -36,14 +36,25 @@ PATH="${GOBIN}:${PATH}"
 export GO111MODULE=on
 
 # Install golangci-lint
-echo 'installing golangci-lint '
+echo "installing golangci-lint and logcheck plugin from hack/tools into ${GOBIN}"
 pushd "${KUBE_ROOT}/hack/tools" >/dev/null
   go install github.com/golangci/golangci-lint/cmd/golangci-lint
+  go build -o "${GOBIN}/logcheck.so" -buildmode=plugin k8s.io/klog/hack/tools/logcheck/plugin
```

Comment:
I still don't think we should be consuming/compiling a hack tool from klog this way, but this PR is just relocating the existing use, so it's not making things *worse*

Reply:
I'm fine with moving it to a separate repo. Tracked in https://github.com/kubernetes/klog/issues/312.

Reply:
agreed